src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/adapter/EnhetsregisteretAdapter.kt

```diff
@@ -45,18 +45,21 @@ class EnhetsregisteretAdapter(private val appProperties: AppProperties) {
 
     private fun EnhetsregisteretOrganization.withOrgPath(): EnhetsregisteretOrganization {
         val idSet: MutableSet<String> = mutableSetOf(organisasjonsnummer)
+        val isTestOrganization = appProperties.testOrganizations.contains(organisasjonsnummer)
```

Comment:
Ta med en sjekk på om du faktisk er i et testmiljø. appProperties.organizationCatalogueHost med contains for localhost, staging og demo burde vel fungere?

Reply:
Rettet